CMakeLists.txt

```diff
@@ -784,6 +784,11 @@ if(NOT MSVC)
   string(APPEND CMAKE_CXX_FLAGS " -Wall")
   string(APPEND CMAKE_CXX_FLAGS " -Wextra")
   string(APPEND CMAKE_CXX_FLAGS " -Werror=return-type")
+  if(NOT USE_CUDNN)
```

Comment:
For this flag, I will create a follow-up task to fix the issue upstream in cudnn_frontend where the warning comes from. After that is fixed, I think we can remove this not-so-good check

Reply:
But can we perhaps suppress one or two files like that?

Reply:
FYI, PR on the 3rd party cudnn_frontend https://github.com/NVIDIA/cudnn-frontend/pull/33. When this is accepted and merged, this check can be removed